### ISSUE TITLE
elm-test in root package.json

### ISSUE THREAD

**Issue by sporto** [NoRedInk/json-elm-schema#34]
This packages has elm-test in the root elm-package.

This is not necessary as it should be in test/elm-package.

Also having this dependency here prevents me from installing this package as it conflicts with other packages that also have elm-test in the root with a different version. e.g. https://github.com/lukewestby/elm-string-interpolate

Please remove elm-test from the root `elm-package.json`

**Comment by ericgj** [comment]
I noticed that too, and elm-test brings in a bunch of other dependencies. But I think it is needed because of the fuzzers. I can't speak for the maintainers, but to me it would make sense to move JsonSchema.Fuzz to a separate library for those who don't need the fuzz testing functionality (nice as it is!)

**Comment by jwoudenberg** [comment]
Yeah, it's the fuzzers. I wonder if widening the version range of allowed `elm-tests`'s would address your problem? Pulling out the Fuzzers into a separate library would be a solution too, but quite a bit more involved because we'd need to expose more of the internals of the json schema library to make it possible.
